todo_app/templates/trello.html

```diff
@@ -18,12 +18,21 @@ <h1 class="display-4">Trello App</h1>
     </form>
   <div class="row">
     <div class="col-md-12">
-      <h2>Cards</h2>
+      <h2>To Do Cards</h2>
       <ul class="list-group mb-4">
-        {% for item in items %}
+        {% for item in view_model_to_do.items %}
```

Comment:
You need to remove `.items` from here for this to work because `view_model_to_do` is already just a list of items.

But actually the intended usage of the view model described in the exercise is to have a single view model that you pass in from app.py with e.g. `return render_template('trello.html', view_model=view_model)` and then you would change this line to `for item in `view_model.to_do_items`

Reply:
Have fixed this issue as well. Though currently am only using one View Model but with filters in the app itself.

Reply:
Have now done it so it is return `render_template('trello.html', view_model=view_model)` and the delination in the trello.html